packages/@aws-cdk/aws-codebuild-artifacts/lib/artifacts.ts

```diff
@@ -0,0 +1,84 @@
+import codebuild = require("@aws-cdk/aws-codebuild");
+import s3 = require("@aws-cdk/aws-s3");
+
+/**
+ * CodePipeline Artifact definition for a CodeBuild Project.
+ * *Note*: this type cannot be used as a secondary artifact,
+ * and because of that, you're not allowed to specify an identifier for it.
+ */
+export class CodePipelineBuildArtifacts extends codebuild.BuildArtifacts {
```

Comment:
omit the words "build" and "artifacts" if they already exist in the package name. The package name/namespace is part of the name of the class. That's why it's @aws-cdk/aws-lambda.Function and not LambdaFunction.

Reply:
How does that work if you want Artifacts to be merged into `@aws-cdk/aws-codebuild-sources`? `codebuild_sources.CodePipeline???`?

Reply:
Also, that's not consistent with how we name other integrations, for example `codepipeline_actions.GitHubSource**Action**`.

Reply:
In the API reviews, we actually just said we wanted to remove the "Action" postfix from codepipeline_actions

Reply:
Let's just leave sources in the codebuild module...